mace/data/utils.py

```diff
@@ -120,6 +123,14 @@ def config_from_atoms(
     stress = atoms.info.get(stress_key, None)  # eV / Ang
     virials = atoms.info.get(virials_key, None)
     dipole = atoms.info.get(dipole_key, None)  # Debye
+    group = atoms.arrays.get(group_key, None)
```

Comment:
can we use the `tags` property which is already present by default in `ase.atoms` instead of creating a new one? 
https://wiki.fysik.dtu.dk/ase/ase/atoms.html#ase.Atoms.get_tags

Reply:
We can do this, but I think using the generic `atoms.arrays.get` method gives us greater flexibilities. When you want to keep the `tag` data field in your dataset, using another data array should be useful.

Besides, setting data array in ASE is simple enough. You could just do somethings like `atoms.set_array('charges', [0] * 10 + [1] * 20, int, ())`